### ISSUE TITLE
Invalid "extends" eslint config option inside "overrides" entry

### ISSUE THREAD

**Issue by gnapse** [kentcdodds/static-testing-tools#11]
This happens in the code for the lesson to add support for eslint to support TypeScript files. In that lesson there's an instruction to add a "extends" config inside the "overrides" section intended to tailor the eslint config for TypeScript files.

Here's the exact line: https://github.com/kentcdodds/static-testing-tools/blob/28545516e33a63b3ea98eacfd0597b4ed8d8048b/.eslintrc#L24

**Comment by 5tormTrooper** [comment]
Having the same issue. Any fix?

**Comment by kentcdodds** [comment]
Yeah, this was a mistake on my part. That feature is not supported. I'll be updating the course later this year most likely

**Comment by 5tormTrooper** [comment]
Until then, is there a work-around for making eslint support typescript files?

**Comment by kentcdodds** [comment]
This was updated in November (for 2020). There wasn't an error before, but things move fast and change a lot in the JavaScript ecosystem.

I'm getting this error now as well (despite `package-lock.json` 🙄).

After digging a little bit, I found this workaround: https://github.com/typescript-eslint/typescript-eslint/issues/890

Basically, add `"createDefaultProgram": true` to your `parserOptions` in `.eslintrc` file. It's not recommended, but it's the workaround suggested by the team at the moment. Hopefully when I circle back on this to update things, we'll have a more solid solution.

I will be updating things later this year and we'll be in a good place again.
